testimony/__init__.py

```diff
@@ -200,7 +209,7 @@ def __str__(self):
         return '\n'.join(output)
 
 
-def main(report, paths, json_output, nocolor):
+def main(report, paths, json_output, nocolor, tags=[]):
```

Comment:
Using mutable objects as default value is not a good practice, change `tags=None` and check if it is none. Or even drop the default value.